packages/apps/graph/src/scripts/devnet/simulate.ts

```diff
@@ -161,6 +165,6 @@ export async function simulate({
     }
     counter++;
     // Timeout
-    await new Promise((resolve) => setTimeout(resolve, transferInterval));
+    await new Promise((resolve) => setTimeout(resolve, duration));
```

Comment:
This will ensure that when it loops through the length of accounts, it will timeout for the given total duration of the simulation. I am not sure this is the intended behaviour 😅 
```suggestion
    await new Promise((resolve) => setTimeout(resolve, transferInterval));
```